chingsley/ireporter
# implement fetch to consume api for user login

#### What does this PR do?
This pull request deals with implementation of user login by using fetch to consume the API endpoint.

#### Description of Task to be completed?
- create the login.js file to implement the fetch
- using fetch consume the API with the route [host]/api./v1/auth/login
- create a dialog box to handle and display error messages
- on success redirect the user to either user report history or admin 'manageorders' page depending on the user status.

#### How should this be manually tested?
- clone the repo
- cd into the root directory
- run `npm install' to install the necessary dependencies
- run `npm test' to see the tests pass
- run `npm start' to start the server
- at the frontend, test the functionalities by signing in using both correct and incorrect credentials

#### What are the relevant pivotal tracker stories
#163056323
